src/components/axes/axis.ts

```diff
@@ -430,19 +413,13 @@ module Plottable {
     }
 
     /**
-     * Gets whether the Axis is currently set to show the first and last
-     * tick labels.
-     *
-     * @returns {boolean} whether or not the last
-     * tick labels are showing.
+     * Gets whether the Axis is currently set to show the first and last tick labels.
```

Comment:
Maybe change to...
"Gets whether the Axis will show the end tick labels."

or maybe replace "end" with "first and last"